backend/resources/participant.py

```diff
@@ -32,6 +32,24 @@ def post(self):
         except ValidationError as err:
             return (err.messages), HTTPStatus.BAD_REQUEST
 
+        if (
+            Participant.query.filter(Participant.email == json_data["email"]).first()
+            is not None
+        ):
```

Comment:
I think that `is not None` could be removed.

Reply:
Sure, I can add some tests.